app/models/grda_warehouse/cas_project_client_calculator/boston.rb

```diff
@@ -441,5 +443,27 @@ def majority_sheltered(client)
         recent_ce_self_report_certification.
         pluck(:client_id)
     end
+
+    private def ongoing_enrollment_enrollment_ids(client)
+      range = (Date.yesterday .. Date.current)
+      client.source_enrollments.select { |m| m.open_during_range?(range) }.
+        map { |en| [en.data_source_id, en.enrollment_id] }.to_set
+    end
+
+    # Any open enrollments 4.11.B CurrentlyFleeing = 1
+    private def currently_fleeing(client)
+      client.source_health_and_dvs.select do |m|
+        m.CurrentlyFleeing == 1 &&
+        [m.data_source_id, m.enrollment_id].in?(ongoing_enrollment_enrollment_ids(client))
+      end.any?
+    end
+
+    # Any open enrollments 4.11.2 DomesticViolenceVictim = 1
+    private def domestic_violence?(client)
```

Comment:
Weird that it is a predicate with a `?` but returns an integer. Maybe add a comment to explain this?